target/cheri-common/op_helper_cheri_common.c

```diff
@@ -725,7 +802,8 @@ void CHERI_HELPER_IMPL(cfromptr(CPUArchState *env, uint32_t cd, uint32_t cb,
         raise_cheri_exception(env, CapEx_SealViolation, cb_exc);
     } else {
         cap_register_t result = *cbp;
-        uint64_t new_addr = cbp->cr_base + rt;
+        uint64_t new_addr = rt;
+        if(!(set_addr)) new_addr +=cbp->cr_base;
```

Comment:
```suggestion
        if (!set_addr)
            new_addr += cbp->cr_base;
```
I generally just run git-clang-format -f before committing. Might not look perfect but at least it's usually consistent.